x/auth/client/cli/tx_sign.go

```diff
@@ -19,7 +19,6 @@ const (
 	flagMultisig        = "multisig"
 	flagOverwrite       = "overwrite"
 	flagSigOnly         = "signature-only"
-	flagAmino           = "amino"
```

Comment:
Does this effect ledger signing via cli?

Reply:
I don't think so. I think it's for generating a StdTx which is no longer needed, but it would be great if someone else can confirm

Reply:
See lines 422-439 deleted below - that generates a `StdTx`. I'm not sure where one would submit that except to a legacy chain. If so, they should have a legacy binary to do that. So I think safe to remove

Reply:
@alexanderbez does that seem right to you?

Reply:
Imo we should add a small cli breaking change in changelog for that.

Reply:
Added changelog entries